.github/workflows/backend-api-spec-update.yml

```diff
@@ -9,11 +9,11 @@ jobs:
   update_backend_api_spec:
     runs-on: ubuntu-latest
     steps:
-      - uses: peterjgrainger/action-create-branch@v2.2.0
+      - uses: peterjgrainger/action-create-branch@v2.3.0
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          branch: '/refs/heads/infra/update-backend-api-spec'
```

Comment:
didn't know why it appears extra `/` in action
remove `refs/heads` at all, since it worked fine without it